CrowdFundingApplication/wwwroot/js/site.js

```diff
@@ -27,4 +27,12 @@ var loadFile = function (event) {
     image.src = URL.createObjectURL(event.target.files[0]);
 };
 
-CKEDITOR.replace('FullDescription');
\ No newline at end of file
+CKEDITOR.replace('FullDescription');
+
+var message = function () {
+    alert("Проект был успешно создан и отправлен на модерацию! Статус проекта можно просмотреть в профиле.");
+}
+
+$("#id_message").bind("click", function () {
```

Comment:
And if the validation fails, the message will still be shown?

Reply:
Has been changed